yarn run spawn fails with no libzmq.a

When I run `yarn run spawn` I get the following error (I have `zeromq` and `zmq` and `czmq` installed):

```
clang: error: no such file or directory: './Release/../../zmq/lib/libzmq.a'
make: *** [Release/zmq.node] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/elijahmeeks/.nvm/versions/node/v8.10.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:258:23)
gyp ERR! stack     at emitTwo (events.js:126:13)
gyp ERR! stack     at ChildProcess.emit (events.js:214:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
gyp ERR! System Darwin 17.7.0
gyp ERR! command "/Users/elijahmeeks/.nvm/versions/node/v8.10.0/bin/node" "/Users/elijahmeeks/.nvm/versions/node/v8.10.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/elijahmeeks/sites/nteract/applications/desktop/node_modules/jp-kernel/node_modules/zeromq
gyp ERR! node -v v8.10.0
gyp ERR! node-gyp -v v3.6.2
gyp ERR! not ok 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zeromq@4.6.0 install: `node scripts/prebuild-install.js || (node scripts/preinstall.js && node-gyp rebuild)`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zeromq@4.6.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/elijahmeeks/.npm/_logs/2018-10-08T19_33_14_360Z-debug.log

    at ChildProcess.childProcess.once.code (/Users/elijahmeeks/sites/nteract/node_modules/builder-util/src/util.ts:254:14)
    at Object.onceWrapper (events.js:317:30)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Socket.stream.socket.on (internal/child_process.js:346:11)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at Pipe._handle.close [as _onclose] (net.js:567:12)
From previous event:
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
From previous event:
    at /Users/elijahmeeks/sites/nteract/node_modules/app-builder-lib/src/util/yarn.ts:157:27
From previous event:
    at rebuild (/Users/elijahmeeks/sites/nteract/node_modules/app-builder-lib/out/util/yarn.js:239:18)
    at /Users/elijahmeeks/sites/nteract/node_modules/app-builder-lib/src/util/yarn.ts:20:11
From previous event:
    at installOrRebuild (/Users/elijahmeeks/sites/nteract/node_modules/app-builder-lib/out/util/yarn.js:68:17)
    at /Users/elijahmeeks/sites/nteract/node_modules/electron-builder/src/cli/install-app-deps.ts:56:9
    at Generator.next (<anonymous>)
From previous event:
    at installAppDeps (/Users/elijahmeeks/sites/nteract/node_modules/electron-builder/out/cli/install-app-deps.js:174:17)
    at then (/Users/elijahmeeks/sites/nteract/node_modules/electron-builder/src/cli/cli.ts:42:48)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
From previous event:
    at Object.args [as handler] (/Users/elijahmeeks/sites/nteract/node_modules/electron-builder/src/cli/cli.ts:42:48)
    at Object.runCommand (/Users/elijahmeeks/sites/nteract/node_modules/yargs/lib/command.js:238:44)
    at Object.parseArgs [as _parseArgs] (/Users/elijahmeeks/sites/nteract/node_modules/yargs/yargs.js:1059:30)
    at Object.get [as argv] (/Users/elijahmeeks/sites/nteract/node_modules/yargs/yargs.js:1000:21)
    at Object.<anonymous> (/Users/elijahmeeks/sites/nteract/node_modules/electron-builder/src/cli/cli.ts:25:28)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! nteract@0.11.9 install:app-deps: `electron-builder install-app-deps`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the nteract@0.11.9 install:app-deps script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/elijahmeeks/.npm/_logs/2018-10-08T19_33_14_399Z-debug.log
error Command failed with exit code 1.

lerna ERR! yarn run spawn exited 1 in 'nteract'
error Command failed with exit code 1.
```

--- a/applications/desktop/package.json
+++ b/applications/desktop/package.json
@@ -108,7 +108,7 @@
   },
   "dependencies": {
     "@nteract/examples": "3.0.0",
-    "ijavascript": "^5.0.17",
+    "ijavascript": "lgeiger/ijavascript",
     "jmp": "^2.0.0",
     "mathjax-electron": "^2.0.1",
     "nteract-assets": "^4.0.0"

